dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -819,7 +819,21 @@ ShowStatusPropertyDialog(
         else if (pProperties->Status == NCS_MEDIA_DISCONNECTED || pProperties->Status == NCS_DISCONNECTED ||
                  pProperties->Status == NCS_HARDWARE_DISABLED)
         {
-            ShowNetConnectionProperties(pContext->pNet, pContext->hwndDlg);
+            WCHAR buffer[100];
+            
+            LoadStringW(netshell_hInstance, IDS_NETWORKCONNECTION, buffer, _countof(buffer));
+            HWND prophwnd = FindWindow(NULL, buffer);
```

Comment:
usual win32 code style would name this variable:
```suggestion
            HWND hWndProp = FindWindow(NULL, buffer);
```